client/components/mma/cancel/cancellationSaves/supporterplus/SupporterPlusOffer.tsx

```diff
@@ -175,6 +175,14 @@ export const SupporterPlusOffer = () => {
 		'yyyy-MM-dd',
 	).dateStr(DATE_FNS_LONG_OUTPUT_FORMAT);
 
+	const strikethroughPrice = routerState.nonDiscountedPayments.reduce(
+		(prev, current) =>
+			prev && prev.amount > current.amount ? prev : current,
+	).amount;
```

Comment:
Not that I'm not a fan of `reduce` (I am, and think I probably overuse it) but what about this as an alternative? Maybe it's slightly clearer what the intention is?

```suggestion
	const strikethroughPrice = Math.max(...routerState.nonDiscountedPayments.map(p => p.amount));
```

Or splitting it into two:

```js
    const allAmounts = routerState.nonDiscountedPayments.map(p => p.amount);
    const strikeThroughPrice = Math.max(...allAmounts);
```

Reply:
yeah I like this 👍